elrond-wasm-debug/src/api/crypto_api_mock.rs

```diff
@@ -53,8 +54,18 @@ impl CryptoApiImpl for DebugApi {
         panic!("verify_bls not implemented yet!")
     }
 
-    fn verify_ed25519(&self, _key: &[u8], _message: &[u8], _signature: &[u8]) -> bool {
-        panic!("verify_ed25519 not implemented yet!")
+    fn verify_ed25519(&self, key: &[u8], message: &[u8], signature: &[u8]) -> bool {
```

Comment:
There is a test in basic-features that tests this, you should also uncomment that

Reply:
done